.github/workflows/ci_linux_x64_clang.yml

```diff
@@ -28,37 +28,52 @@ jobs:
   linux_x64_clang:
```

Comment:
I added `ci-exactly: linux_x64_clang` to the PR description so this is the only workflow that will run. See https://iree.dev/developers/general/contributing/#ci-behavior-manipulation for more info on that.

(Not strictly needed, but will save some CI time by not running unrelated workflows while you test)